core/lib/object_store/src/objects.rs

```diff
@@ -102,15 +109,14 @@ impl StoredObject for SnapshotStorageLogsChunk {
 
     fn encode_key(key: Self::Key<'_>) -> String {
         format!(
-            "snapshot_l1_batch_{}_storage_logs_part_{:0>4}.json.gzip",
+            "snapshot_l1_batch_{}_storage_logs_part_{:0>4}.proto.gzip",
             key.l1_batch_number, key.chunk_id
         )
     }
 
-    //TODO use better language agnostic serialization format like protobuf
     fn serialize(&self) -> Result<Vec<u8>, BoxedError> {
```

Comment:
Future work (?): Given that (de)serialization for the new structs is computationally expensive, I wonder whether `serialize` / `deserialize` should be made `async` so that they will move computations to a blocking thread. As implemented currently, these methods risk blocking the Tokio reactor for an extended period of time (probably not an issue given the current usage patterns, just something we should keep in mind).

Reply:
I think this change could positively (though probably marginally) impact our use case as well as no new queries can be sent when anything is serialized, I'll keep this optimization in mind